core/src/main/java/org/frankframework/receivers/ResourceLimiter.java

```diff
@@ -21,27 +21,19 @@
  * See {@link java.util.concurrent.Semaphore} for generic details.
  */
 public class ResourceLimiter extends java.util.concurrent.Semaphore {
-	private int maxResourceCount;
 
-	public ResourceLimiter(int permits, int maxResourceCount) {
+	public ResourceLimiter(int permits) {
```

Comment:
The initial nr of permits and the maxResourceCount should be the same, instead of 2 different parameters to the constructor.
That was the idea of the resource-limiter that you can ask what is the max nr of resources allowed, and also use to actually limit the nr of resources used (via the semaphore-methods).